quickjs.h

```diff
@@ -631,6 +632,11 @@ JS_EXTERN void JS_FreeValueRT(JSRuntime *rt, JSValue v);
 JS_EXTERN JSValue JS_DupValue(JSContext *ctx, JSValue v);
 JS_EXTERN JSValue JS_DupValueRT(JSRuntime *rt, JSValue v);
 JS_EXTERN int JS_ToBool(JSContext *ctx, JSValue val); /* return -1 for JS_EXCEPTION */
+static inline JSValue JS_ToBoolean(JSContext *ctx, JSValue val)
```

Comment:
Is this one liner really necessary?

Reply:
It nicely mirrors the `ToBoolean()` abstract operation (https://262.ecma-international.org/#sec-toboolean) which we use for type coercion: https://github.com/holepunchto/libqjs/blob/d32c3f5c95dfb4de30c54d86aaee498c6886d914/src/js.c#L3065-L3080

Reply:
Good point about the abstract operation 👍